pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_improving-search-results-with-abstract.adoc

```diff
@@ -0,0 +1,15 @@
+:_module-type: CONCEPT
+
+[id="con_improving-search-results-with-abstract_{context}"]
+= Role of abstract in search engine results
+
+[role="_abstract"]
+In {ProductShortName}, each source file includes an abstract that provides search engines with metadata about your content. The abstract describes the purpose of document and also helps the reader to determine whether the content is useful to them.
+
+{ProductShortName} automatically detects the first paragraph of a module or an assembly as the abstract. However, you can add the `[role="_abstract"]` tag in the source files to specify the paragraph that you want to use as an abstract. Ensure that the abstract immediately follows the tag, with no blank lines in between. The `[role="_abstract"]` tag is optional.
+
+Use consistent phrasing in abstracts across documents to make your content easy to find. You can include simple phrases and keywords that your readers might enter into a search engine. The abstract is displayed in the Red Hat Customer Portal search results and Google Search results.
```

Comment:
`...and Google Search results.`
What about other search engines? Unless you know something I don't, I wouldn't want to single out Google in EUD. Practically speaking, that's might be what we care about most, but still, I don't think like we should be showing preference.

Reply:
Yes, you're right. I changed that to web search engine